### PR TITLE
fix dialog title, remove red border from correction todo and disable …

### PR DESCRIPTION
**Pull request by BrandonSharratt**
…change when a correction filing is in todo

*Issue:*https://github.com/bcgov/entity/issues/23948
https://github.com/bcgov/entity/issues/23472

*Description of changes:*
Removed red border error style from Correction to do items, disabled change address / director buttons when a correction draft/pending exists. fixed dialog title for dissolution

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-60-9h22rajc.web.app

File: src/pages/dashboard.vue [hunk 1]
```diff
@@ -186,8 +186,24 @@ const pendingAddress = computed(() => {
...
+    return corrections.length > 0
+  }
+  return false
+})
+
+const isChangeAddressDisabled = computed(() =>
+  business.currentBusiness.adminFreeze || pendingAddress.value || hasProgressCorrections.value
+)
+
+const isChangeDirectorDisabled = computed(() => business.currentBusiness.adminFreeze || hasProgressCorrections.value)
```
Comment (patrickpeinanw) [inline src/pages/dashboard.vue:206]: I think this ticket is only for the red border in the todo section. The 'Change' button in the address section will be handled in #24010. The screenshot shows all the mismatches, but we broke the original QA bug into different ticket. 

Also, the logic for `isChangeAddressDisabled` and `isChangeDirectorDisabled` seems wrong. 

https://github.com/bcgov/business-filings-ui/blob/3a62b2b470bf16c181b879057823ff795e5faf68/src/views/Dashboard.vue#L131-L145  it uses: `:disabled="!isAllowed(AllowableActions.ADDRESS_CHANGE)"`. The function `isAllowed()` is already implemented in the business store.

Comment (BrandonSharratt) [reply src/pages/dashboard.vue:206]: The ticket I grabbed mentioned the change buttons still so I added handling for the case when a correction filing exists. (It felt large for a 1) The logic in isChangeAddressDisabled and isChangeDirectorDisabled was already there I just added the hasProgressCorrections flag

Comment (BrandonSharratt) [reply src/pages/dashboard.vue:206]: We will handle the other cases in #24010

**Review by patrickpeinanw — APPROVED** [review]

**Pull request merged**